src/ports/tabby/templates/tabby-colors.yaml

```diff
@@ -0,0 +1,52 @@
+terminal:
+  searchOptions: {}
+  colorScheme:
+    name: {{ theme_name }}
+    foreground: '{{ accent7 }}'
+    background: '{{ accent12 }}'
+    selection: '{{ accent20 }}'
```

Comment:
Have you tested on Tabby after this change of selection to accent20? Worked well?